src/docs/guides/24-how-to-use-time-based-bookings/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: How to take time-based bookings into use
+slug: how-to-take-time-based-bookings-into-use
+updated: 2019-04-10
+category: guides
+ingress:
+  Time-based bookings and availability management enable low level fine
+  tuning of possible booking models for listings.
+published: true
+---
+
+## 1. Process change
```

Comment:
This is quite a low level description of the required process adjustment but as we'll soon be developing the CLI I think it makes sense to open the internals of the process a bit.